insonmnia/worker/overseer.go

```diff
@@ -33,7 +32,7 @@ const dieEvent = "die"
 // Description for a target application.
 type Description struct {
 	pb.Container
-	Reference    reference.Reference
+	Reference    string
```

Comment:
why?

Reply:
Has to be marshalled to be saved to storage, but not really clear how to marshal this interface.

Reply:
I suggest to provide custom marshaller for Description, instead of using primitives